specification/gedcom-03-datamodel.md

```diff
@@ -1018,12 +1018,6 @@ Each `NOTE`.`TRAN` must have either a `MIME` or `LANG` substructure, and may hav
 
 See `SHARED_NOTE_RECORD` for advice on choosing between `NOTE` and `SNOTE`.
 
-:::note
-Although notes may be tagged with the language that they are written in, this specification does not provide a mechanism for distinguishing between notes with distinct content in distinct languages versus notes with the same content translated into distinct languages. It is expected that a future version of this specification will provide a mechanism for a single note to have multiple language translations.
```

Comment:
Upon re-reading, I think the first sentence of this note is still valid.   That is, LANG is still {0:1} so if the NOTE <Text> contains a mix of two languages, there is no way for distinguishing which two languages are in the text.

Reply:
I don't think that's what the note means. We could add a note like that, but this note is not that note.